packages/podman-extension-api/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@podman-desktop/podman-extension-api",
+  "version": "0.0.1",
+  "description": "API for Podman Desktop Podman extension",
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "types": "./src/podman-extension-api.d.ts",
+  "files": [
+    "src"
+  ],
+  "scripts": {
+    "prepare": "",
+    "clean": "rimraf lib *.tsbuildinfo",
+    "build": "",
+    "watch": "",
+    "publish:next": "yarn publish  --registry=https://registry.npmjs.org/ --no-git-tag-version --new-version 0.0.1-\"$(date +%s)\""
+  },
+  "dependencies": {
+    "@podman-desktop/api": "0.0.1"
```

Comment:
Why would we need to include "@podman-desktop/api" ?

Reply:
See imports in d.ts file/